ocs_ci/ocs/ui/base_ui.py

```diff
@@ -2249,8 +2303,8 @@ def close_browser():
 
     """
     logger.info("Close browser")
-    take_screenshot()
-    copy_dom()
+    take_screenshot("close_browser")
+    copy_dom("close_browser")
     SeleniumDriver().quit()
     SeleniumDriver.remove_instance()
     time.sleep(10)
```

Comment:
Same as above, we should avoid any redundant sleeps if not needed.

Reply:
this comment addressed to the code committed and merged 22/01/2023. lets do review to the code which is on review